src/schema/submission.yaml

```diff
@@ -704,45 +727,43 @@ classes:
           A short description for the Data Access Policy.
         required: true
       policy_text:
-        description: >-
-          The terms of data use and policy verbiage should be captured here.
         required: true
       policy_url:
-        description: >-
-          URL for the policy, if available. This is useful if the terms of the policy is
-          made available online at a resolvable URL.
         required: false
         recommended: true
-      data_access_committee:
-        description: >-
-          The Data Access Committee linked to this policy.
-        required: true
-        inlined: false
       data_use_permission:
         description: >-
-          Data use permission associated with a policy. Typically one or more terms from DUO
+          Data use permission associated with a Data Use Policy. Typically one or more terms from DUO
           and should be descendants of 'DUO:0000001 data use permission'.
-          Please use 'DUO:0000026' if no other permission applies.
+          Please use 'DUO:0000006' if no other permission applies.
```

Comment:
DataUsePermissionEnum defines the permissible values that are not in the format of an ontology ID. If we want the user to enter the value, we should replace DUO:0000006 with its value.